Project1/Unit.h

```diff
@@ -14,10 +14,16 @@
 #include "Experience.h"
 #include "Mobility.h"
 #include "Health.h"
+#include "PartyBase.h"
 
+class PartyBase;
 using IDENTITY = std::string;
 using CONTEXT = Dice<100>;
-
+struct UnitData {
```

Comment:
Why? Just to make construction easier?

Reply:
Yeah.
It also makes it easier to construct a bunch from a file (like when initializing a party )